### PR TITLE
Fix: Update url in the downloader and tests

### LINKED ISSUE #90: Google chrome driver endpoint changed.
The Downloader for the `ChromeDriver` uses `https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing` as endpoint for the downloads.

Google seems to have changed this domain to `https://storage.googleapis.com/chrome-for-testing-public/`.

A quick fix would  be to just change the download endpoint. But possibly we should look into the option to resolve the download url, from the page we use to resolve the versions?

Edit:
This change seems to have been made between patches `121.0.6166` and `121.0.6167`

### PR DESCRIPTION
**Pull request by KevinVanSonsbeek**
Fixes #90

### CODE REVIEW

File: src/Driver/ChromeDriver/Downloader.php [hunk 1]
```diff
@@ -35,7 +35,7 @@ final class Downloader implements DownloaderInterface
     private const BINARY_LINUX           = 'chromedriver_linux64';
     private const BINARY_MAC             = 'chromedriver_mac64';
     private const BINARY_WINDOWS         = 'chromedriver_win32';
-    private const DOWNLOAD_ENDPOINT_JSON = 'https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing';
+    private const DOWNLOAD_ENDPOINT_JSON = 'https://storage.googleapis.com/chrome-for-testing-public';
```
Comment (dbrekelmans) [inline src/Driver/ChromeDriver/Downloader.php:38]: Did the download URL change for previous versions as well, or are those still on `edgedl.me.gvt1.com`?
We'd want the download for all versions to still work, not just the latest version.

I like your suggestion in #90:
> But possibly we should look into the option to resolve the download url, from the page we use to resolve the versions?

Comment (KevinVanSonsbeek) [reply src/Driver/ChromeDriver/Downloader.php:38]: The endpoint just returns a 404 now, as it seems to have been taken down completely.

I'll quickly have a look to see if i can find an endpoint from an older version, and see if that returns a 404 as well. 

But https://googlechromelabs.github.io/chrome-for-testing/ just mentions the new URL now. For stable, beta, dev and canary

Comment (KevinVanSonsbeek) [reply src/Driver/ChromeDriver/Downloader.php:38]: Oof. It seems only the new ones have actually moved. Which means we will need to add an additional check, with a third endpoint :/

Comment (dbrekelmans) [reply src/Driver/ChromeDriver/Downloader.php]: I'll merge this as a patch version. Would be nice to get a follow-up PR if you're willing to resolve the URL in the same way as the versions.

**Pull request merged**